src/rest/rest-api.php

```diff
@@ -8,15 +13,30 @@
 
 defined( 'WPCOMVIP__BLOCK_DATA_API__PARSE_TIME_ERROR_MS' ) || define( 'WPCOMVIP__BLOCK_DATA_API__PARSE_TIME_ERROR_MS', 500 );
 
+/**
+ * Rest API that will be used to fetch block data.
+ */
 class RestApi {
+	/**
+	 * Initialize the Rest API class.
+	 * 
+	 * @access private
+	 */
 	public static function init() {
 		add_action( 'rest_api_init', [ __CLASS__, 'register_rest_routes' ] );
 	}
 
+	/**
+	 * Validate the block names that have been provided, to ensure that they are valid
+	 * 
+	 * @param string $param the block names to validate.
+	 * 
+	 * @return bool true, if they are valid or false otherwise
+	 */
 	public static function validate_block_names( $param ) {
 		$block_names = explode( ',', trim( $param ) );
 
-		// Validate that all block names are valid
+		// Validate that all block names are strings and are in the format of namespace/block-name.
```

Comment:
Whoops, just saw this clarification after [the comment above](https://github.com/Automattic/vip-block-data-api/pull/43/files#r1281232483). If we include that information in the function comment, we can probably remove it here.